### PR TITLE
Support Laravel 9 LTS

### PR DESCRIPTION
**Pull request by LukeTowers**
See https://github.com/wintercms/winter/pull/148 for more information.

### CODE REVIEW

File: src/Foundation/Application.php [hunk 1]
```diff
@@ -70,7 +70,7 @@ public function publicPath()
      *
      * @return string
      */
-    public function langPath()
+    public function langPath($path = '')
     {
         return PathResolver::join($this->basePath, '/lang');
```
Comment (mjauvin) [inline src/Foundation/Application.php]:
```suggestion
        return PathResolver::join($this->basePath, '/lang' . (!empty($path) ? "/$path" : ''));
```

File: src/Auth/Manager.php [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <?php namespace Winter\Storm\Auth;
 
-use Cookie;
-use Session;
-use Request;
+use \Illuminate\Support\Facades\Cookie;
+use \Illuminate\Support\Facades\Session;
+use \Illuminate\Support\Facades\Request;
```
Comment (LukeTowers) [inline src/Auth/Manager.php]:
```suggestion
use Illuminate\Support\Facades\Cookie;
use Illuminate\Support\Facades\Session;
use Illuminate\Support\Facades\Request;
```

File: src/Halcyon/Datasource/ResolverInterface.php [hunk 3]
```diff
@@ -1,28 +1,48 @@
...
 {
-
     /**
-     * Get a datasource instance.
+     * Get a datasource instance by name.
      *
-     * @param  string  $name
+     * @param string $name The name of the datasource to retrieve.
      * @return \Winter\Storm\Halcyon\Datasource\DatasourceInterface
+     * @throws \Winter\Storm\Halcyon\Exception\MissingDatasourceException If a datasource with the given name does not exist.
```
Comment (LukeTowers) [inline src/Halcyon/Datasource/ResolverInterface.php]:
```suggestion
     * @throws \Winter\Storm\Halcyon\Exception\MissingDatasourceException If a datasource with the given name does not exist.
```

**Comment by LukeTowers** [comment]
**NOTE** v1.1.9 will need to be reviewed and made live before this PR is merged into develop.

**Pull request merged**